### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by fgeierst** [fgeierst/warumichhierbin#8]
- [x] Background and foreground colors do not have a sufficient contrast ratio
- [x] `<iframe>` element does not have a title

**Comment by fgeierst** [comment]
Text color contrast does still not meet WCAG requirements. But introducing the darker `--color-primary-text: #00a9ac;` has at least improved readability.
